src/meta-srv/Cargo.toml

```diff
@@ -61,7 +61,7 @@ url = "2.3"
 
 [dev-dependencies]
 chrono.workspace = true
-client = { workspace = true, features = ["testing"] }
```

Comment:
another question about this: because we already have `testing` feature in `client`, is it really necessary to move `database.rs` out of `client`?

Reply:
and we do have only one testing function in client for which we can't remove `testing` from `client` at the moment.

Reply:
I agree we can move this struct to `client` crate behind the `testing` feature.

IIRC previously it's in the `client` crate without `testing` feature in control.

Reply:
I think we can close this one, revert #3820, and create a new patch to feature-gate `database.rs`

Reply:
Yeah. This is possible. Would you follow these changes?